<?xml version="1.0"?>
<ui version="4.0">
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Form</class>
 <widget class="QGroupBox" name="groupControl">
       <property name="title">
        <string>Control</string>
       </property>
       <layout class="QGridLayout" name="controlGrid">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushScan">
          <property name="toolTip">
           <string>Scan indefinitely without recording</string>
          </property>
          <property name="text">
           <string>Scan</string>
          </property>
          <property name="iconSize">
           <size>
            <width>8</width>
            <height>12</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushAcquire">
          <property name="toolTip">
           <string>Scan as defined by </string>
          </property>
          <property name="text">
           <string>Acquire</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
 <pixmapfunction></pixmapfunction>
 <connections/>
</ui>
